### DISCUSSION TITLE
how to make whole terminal readonly ?

### DISCUSSION THREAD

**Discussion by HeiShuXianSheng** [sourcelair/xterm.js#4702] [Q&A] [answered]
I want to open xterm with two step as blow:
 step1: open xterm and print something in screen;
 step2: make whole xterm readonly, this way the cursor won't blink and it won't receive input from keyboard keys;

I have finished the first step, but how can i get a readonly xterm?

**Comment by tisilent** [comment, marked as answer]
`term.option.disableStdin`

**Reply by HeiShuXianSheng** [reply]
thanks

**Reply by tisilent** [reply]
options  💄 . 😭  ..

**Comment by tisilent** [comment]
`term.write("\x1b[?25l");`
Cursor not visible.
